docs/variants/dell_optiplex/building-manual.md

```diff
@@ -2,23 +2,42 @@
 
 **Please read the [overview page](overview.md) first!**
 
-To build Dasharo compatible with Dell OptiPlex 7010/9010, follow the steps
-below:
+## Available variants
+
+To build Dasharo compatible with Dell OptiPlex 7010/9010, you need to decide
+what is your desired configuration. The available options are as follows:
+
+- Dasharo (coreboot + SeaBIOS)
+- Dasharo (coreboot + edk2)
+- Dasharo (coreboot + SeaBIOS) with
+  [Intel TXT](https://doc.coreboot.org/security/intel/txt.html) support
+- Dasharo (coreboot + edk2) with
+  [Intel TXT](https://doc.coreboot.org/security/intel/txt.html) support
+
+### Intel TXT
+
+If your choice is to enable TXT support, be advised that there are proprietary
+[ACM blobs](https://doc.coreboot.org/security/intel/acm.html) required for the
+firmware to work properly. They are non-redistributable for the platform in
+question, which means you will need to obtain/extract them yourself and patch
+the result binary using `cbfstool`. The methods will not be covered in this
+guide.
```

Comment:
I would redirect to the initial deployment or something like that. Initial deployment page should have it described

Reply:
Added a guide on blob patching to initial deployment in [this commit](https://github.com/Dasharo/docs/pull/873/commits/2c3447ab0aac6ecbe14bd0cf8bd1d80e4bc393a7)